cmd/agent/common/common_windows.go

```diff
@@ -25,29 +25,14 @@ var (
 	viewsPath string
 )
 
-var (
-	// DefaultConfPath points to the folder containing datadog.yaml
-	DefaultConfPath = "c:\\programdata\\datadog"
-	// DefaultLogFile points to the log file that will be used if not configured
-	DefaultLogFile = "c:\\programdata\\datadog\\logs\\agent.log"
-	// DefaultDCALogFile points to the log file that will be used if not configured
-	DefaultDCALogFile = "c:\\programdata\\datadog\\logs\\cluster-agent.log"
-	//DefaultJmxLogFile points to the jmx fetch log file that will be used if not configured
-	DefaultJmxLogFile = "c:\\programdata\\datadog\\logs\\jmxfetch.log"
-	// DefaultCheckFlareDirectory a flare friendly location for checks to be written
-	DefaultCheckFlareDirectory = "c:\\programdata\\datadog\\logs\\checks\\"
-	// DefaultJMXFlareDirectory a flare friendly location for jmx command logs to be written
-	DefaultJMXFlareDirectory = "c:\\programdata\\datadog\\logs\\jmxinfo\\"
-)
-
 func init() {
 	pd, err := winutil.GetProgramDataDir()
 	if err == nil {
-		DefaultConfPath = pd
-		DefaultLogFile = filepath.Join(pd, "logs", "agent.log")
-		DefaultDCALogFile = filepath.Join(pd, "logs", "cluster-agent.log")
+		constants.DefaultConfPath = pd
```

Comment:
I'm worried that with the constants moved to another package that this package might not be imported and the constants won't be updated. Should an `init` be added to `constants_windows.go` and these path updates moved there? It looks like some of the constants aren't updated, too.

Reply:
That makes sense 👍 I'll adjust it 🙇